cpp/oneapi/dal/backend/primitives/intersection/intersection.hpp

```diff
@@ -62,6 +65,30 @@ ONEDAL_FORCEINLINE std::int32_t _popcnt32_redef(const std::int32_t &x) {
     {}
 #endif
 
+#ifdef __ARM_ARCH
```

Comment:
The Arm implementation looks like the default implementation above, so this specialization isn't required at the moment.